gateway/src/apicast/policy/rate_limit/rate_limit.lua

```diff
@@ -61,19 +72,35 @@ local function redis_shdict(url)
   }
 end
 
-local function error(logging_only, status_code)
-  if not logging_only then
-    return ngx.exit(status_code)
+local function error(error_settings, type)
+  if error_settings[type]["error_handling"] == "exit" then
+    return ngx.exit(error_settings[type]["status_code"])
+  end
+end
+
+local function init_error_settings(config_error_settings)
+  local error_settings = default_error_settings
+  if config_error_settings then
+    for _, error_setting in pairs(config_error_settings) do
+      if error_setting.type then
```

Comment:
Do we need this check? I think the type should be required according to the schema.